src/components/Kayttaja/schema.js

```diff
@@ -25,6 +25,7 @@ const schema = {
 
 export const paakayttaja = R.dissoc('henkilotunnus', schema);
 export const patevyydentoteaja = R.dissoc('virtu', schema);
+export const laskuttaja = R.dissoc('henkilotunnus', R.dissoc('virtu', schema));
```

Comment:
This seems like there might be a more elegant way to compose multiple `dissoc` operations.

Reply:
R.omit(['henkilotunnus', 'virtu'])

Reply:
Näyttäisi vastaavan juuri sitä mitä haettiin. Vaihdoin käyttöön.